### ISSUE TITLE
A11y SERIOUS: `auro-counter` disable state contrast on label

### ISSUE THREAD

**Issue by sun-mota** [AlaskaAirlines/auro-formkit#774]
### What browser client(s) are you seeing the issue on?

- [ ] Firefox
- [ ] Chrome
- [ ] Safari
- [ ] Microsoft Edge

### What devices platform(s) are you seeing the issue on?

- [ ] Desktop
- [ ] Mobile

### Steps to reproduce

```
Issue target: auro-counter,#counter-label
      Context: <label id="counter-label" class="label"><slot></slot></label>
      Fix any of the following:
        Element has insufficient color contrast of 1.54 (foreground color: #d0d0d0, background color: #ffffff, font size: 12.0pt (16px), font weight: normal). Expected contrast ratio of 4.5:1
      ---
```

### Reproducible on the Auro doc site

- [ ] I am able to reproduce this issue on the [Auro doc site](https://auro.alaskaair.com).

**Comment by jason-capsule42** [comment]
WCAG criteria states that disabled elements do not need to meet color contrast criteria. Ref: https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html (search for "Inactive User Interface Components") (Thanks @StephenRiosDev)

**Comment by sun-mota** [comment]
aria-disabled should be added to indicate that the element is disabled.

**Comment by blackfalcon** [comment]
> WCAG criteria states that disabled elements do not need to meet color contrast criteria. Ref: https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html (search for "Inactive User Interface Components") (Thanks [@StephenRiosDev](https://github.com/StephenRiosDev))

Please also note that this information has been added to the official Accessibility Checklist under Color Contrast
https://wiki.devtools.teamaag.com/en/standards/accessibility/checklist
